prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -822,6 +826,13 @@ def build_transmission(method="line2sub", **kwargs):
         lambda x: estimate_branch_rating(x, bus["baseKV"]), axis=1
     )
 
+    # Update substation max & min voltages using bus data (from lines)
+    substations.replace(
+        {"MAX_VOLT": -999999, "MIN_VOLT": -999999}, float("nan"), inplace=True
+    )
```

Comment:
Not sure whether we need this or not, `update` will override everything based on the info from `bus` table if there is no `np.NA`, which shouldn't be the base given each substation contains at least one bus?

Reply:
You're right. This was needed before when the update calls used a combination of the bus voltages and the existing `MAX_VOLT`, `MIN_VOLT` values. Will update.

Reply:
Done.